### ISSUE TITLE
Memory leak with BitmapText

### ISSUE THREAD

**Issue by wmike1987** [pixijs/pixi.js#7003]
Updating bitmap text by doing bitmaptextObj.text = "newString" now has a memory leak. I believe this began in v5.3.3 and is still present in the newest rc 5.4.0.

https://www.pixiplayground.com/#/edit/AGAuOUo9ogkEvDnM3vxqf

It appears that every time BitmapText wants to update itself, it creates several new textures (line 410 here: https://github.com/pixijs/pixi.js/blame/v5.3.3/packages/text-bitmap/src/BitmapText.ts), which in turn spawn new events on the base texture.  @GoodBoyDigital, looks like you may be familiar with this section of code. See the below screenshots which show that the _events array on the base texture grows very large with many updates.

![image](https://user-images.githubusercontent.com/1246000/99191858-3328e480-2735-11eb-835c-3ac53b82a4a4.png)

Then after about 20 seconds...
![image](https://user-images.githubusercontent.com/1246000/99191996-f27d9b00-2735-11eb-8e58-0c76134e0876.png)

**Comment by Article19** [comment]
Most of my games use a BitmapText timer that updates about once-per-second. I probably haven't noticed the memory leak since my games only last a few minutes - but this is clearly an issue that needs solving. Cheers!

**Comment by JetLua** [comment]
```js
// Destroy existing texture?
pageMeshData.mesh.texture?.destroy()
pageMeshData.mesh.texture = new Texture(texture.baseTexture);
```

**Comment by bigtimebuddy** [comment]
Would love it if someone could find and verify a fix for this memory leak (bounty attached). @JetLua Suggestion sounds plausible but it would be nice to avoid the constant Texture allocation if possible.

**Comment by Article19** [comment]
Would love to help, but diving into platform source code in this way was never quite my expertise.

I did notice that it mentions in the source that updateText()  "should only be called if the BitmapFont is regenerated." - yet - it seems to be getting called every time the text property of a bitmapText is being set. Seems like a LOT of overhead for setting text...something I'm sure many of us are doing frequently in an app.

I did quickly try JetLua's suggestion of destroying the texture and that does seem to plug the leak. Absent another (better) solution, I wonder if that could be included in 5.4 as a stop-gap measure.
